.github/workflows/update-quick-start-module.yml

```diff
@@ -0,0 +1,107 @@
+name: Update quick start module
+on:
+  schedule:
+    # At 18:30 pm UTC (1:30 pm EST)
+    - cron: "30 18 * * *"
+  pull_request:
+    paths:
+      - .github/workflows/update-quick-start-module.yml
```

Comment:
Shouldn't changes to `_includes/quick-start-module.js` and `_includes/quick_start_local.html` should be included to the list as well?